metis-core/metis-core-service/src/main/java/eu/europeana/metis/core/execution/QueueConsumer.java

```diff
@@ -59,7 +59,7 @@ public void handleDelivery(String consumerTag, Envelope rabbitmqEnvelope,
     if (threadsCounter >= workflowExecutionSettings.getMaxConcurrentThreads()) {
       //Send NACK to send message back to the queue. Message will go to the same position it was or as close as possible
       //NACK multiple(second parameter) we want one. Requeue(Third parameter), do not discard
-      persistenceProvider.getRabbitmqChannel()
+      persistenceProvider.getRabbitmqConsumerChannel()
```

Comment:
User super.getChannel just to be sure you have the right channel? Maybe the super class does something to it... what do you think?